packages/itwinui-css/src/backdrop/backdrop.scss

```diff
@@ -3,9 +3,9 @@
 @import '../style/index';
 
 @mixin iui-backdrop {
+  position: fixed;
```

Comment:
Could you move back where it was so it won't show that you made a change?

Reply:
I moved it